data/so

```diff
@@ -0,0 +1,17 @@
+---
+version: 2
+anglicized_name: Somali
+native_name: Soomaali
+codepoints:
+- !ruby/range 66..68       # B,C,D
+- !ruby/range 70..72       # F,G,H
+- !ruby/range 74..78       # J,K,L,M,N
+- !ruby/range 81..84       # Q,R,S,T
+- !ruby/range 87..89       # W,X,Y
+- !ruby/range 98..100      # b,c,d
+- !ruby/range 102..104     # f,g,h
+- !ruby/range 106..110     # j,k,l,m,n
+- !ruby/range 113..116     # q,r,s,t
+- !ruby/range 119..121     # w,x,y
+- !ruby/range 66688..66717 # 𐒀,𐒁,𐒂,𐒃,𐒄,𐒅,𐒆,𐒇,𐒈,𐒉,𐒊,𐒋,𐒌,𐒍,𐒎,𐒏,𐒐,𐒑,𐒒,𐒓,𐒔,𐒕,𐒖,𐒗,𐒘,𐒙,𐒚,𐒛,𐒜
+- !ruby/range 66720..66729 # 𐒠,𐒡,𐒢,𐒣,𐒤,𐒥,𐒦,𐒧,𐒨,𐒩
```

Comment:
Please remove line 17, those are digits:
```
!ruby/range 66720..66729 # 𐒠,𐒡,𐒢,𐒣,𐒤,𐒥,𐒦,𐒧,𐒨,𐒩
```

Reply:
Also, please change the file name to `so-Osma`, the anglicized & native names to `Somali (Osmanya script)`, and remove the Latin ranges.

Reply:
Changed in https://github.com/typekit/speakeasy/pull/43/commits/11141eb307d0463e1ec44a694faf671288ef0018.